plugins/mobile-plugin/src/main/java/ru/sbtqa/tag/pagefactory/mobile/MobilePage.java

```diff
@@ -1,39 +1,8 @@
 package ru.sbtqa.tag.pagefactory.mobile;
 
-import ru.sbtqa.tag.pagefactory.DefaultPage;
-import ru.sbtqa.tag.pagefactory.annotations.ActionTitle;
-import ru.sbtqa.tag.pagefactory.mobile.utils.SwipeUtils;
-import ru.sbtqa.tag.pagefactory.exceptions.SwipeException;
-import ru.sbtqa.tag.qautils.strategies.DirectionStrategy;
-import ru.sbtqa.tag.qautils.strategies.MatchStrategy;
+import ru.sbtqa.tag.pagefactory.Page;
 
 /**
- * Contains basic ru.sbtqa.tag.pagefactory.mobile.actions in particular with web elements
- * If we want to extend this functional - inherit from this class
+ * Inherit your mobile page objects from this class
  */
-public abstract class MobilePage extends DefaultPage {
-
-    /**
-     * Swipe until text is visible
-     *
-     * @param direction direction to swipe
-     * @param text text on page to swipe to
-     * @throws SwipeException if the text is not found or swipe depth is reached
-     */
-    @ActionTitle("ru.sbtqa.tag.pagefactory.swipe.direction")
-    public void swipeToTextByDirection(String direction, String text) throws SwipeException {
-        SwipeUtils.swipeToText(DirectionStrategy.valueOf(direction.toUpperCase()), text);
-    }
-
-    /**
-     * Swipe until text is visible for Android
-     *
-     * @param strategy contains or exact
-     * @param text text on page to swipe to
-     * @throws SwipeException if the text is not found
-     */
-    @ActionTitle("ru.sbtqa.tag.pagefactory.swipe.match")
-    public void swipeToTextByMatch(String strategy, String text) throws SwipeException {
-        SwipeUtils.swipeToText(MatchStrategy.valueOf(strategy), text);
-    }
-}
\ No newline at end of file
+public abstract class MobilePage implements Page {}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Convert the abstract class "MobilePage" into an interface. ([squid:S1610](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1610))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=96&issues=AWZh46sarFHbA8-Egpmw&open=AWZh46sarFHbA8-Egpmw)

[ISSUE_KEY]: AWZh46sarFHbA8-Egpmw